model/accounts_mgmt/v1/billing_models_resource.model

```diff
@@ -0,0 +1,16 @@
+resource BillingModels {
+    // Retrieves a list of BillingModels.
+	method List {
+		// Index of the requested page, where one corresponds to the first page.
+		in out Page Integer = 1
+
+		// Maximum number of items that will be contained in the returned page.
+		in out Size Integer = 100
+
+		// Total number of items of the collection, regardless of the size of the page.
+		out Total Integer
+
+		// Retrieved list of Capabilities.
+		out Items []BillingModelItem
```

Comment:
You are missing a locator for `BillingModelItem`, and also `billing_model_item_resource.model'

see https://github.com/openshift-online/ocm-api-model/blob/main/model/clusters_mgmt/v1/identity_providers_resource.model
and https://github.com/openshift-online/ocm-api-model/blob/main/model/clusters_mgmt/v1/identity_provider_resource.model
for example

Reply:
Not sure what you mean by a missing locator here, I added an example in ocm-sdk-go and verified that the change is working properly.
Can you explain if I am missing something?

Reply:
As suggested, added locator reference for BillingModel. API for the same is created in AMS : https://gitlab.cee.redhat.com/service/uhc-account-manager/-/merge_requests/4124
Also, working example of the change can be found here: https://github.com/tirthct/ocm-sdk-go/blob/ocm-3077/examples/get_billing_model.go

Reply:
What you added allows getting a billing model by type, but doesn`t allow getting the actual type, 
Also, I wonder why you named it `BillingModelItem` and not `BillingModelType`?

Reply:
> What you added allows getting a billing model by type, but doesnt allow getting the actual type
Can you please explain?

Reply:
Do you mean something like **ocm get /api/accounts_mgmt/v1/billing_models/standard** ?

Reply:
@ciaranRoche I haven't seen any accounts_mgmt classes/structs using the HREF property, so hadn't added it before. Added the HREF property in BillingModelItem and verified with sdk.